wordpress-mobile/gutenberg-mobile
# Correct full width buttons

Fixes the problem with wrongly displayed Buttons on full-width.

gutenberg pr: https://github.com/WordPress/gutenberg/pull/28523

To test:

1. Do sanity testing for Buttons
2. Add Buttons block
3. Fill it with some characters
4. Change the alignment to wide width
5. Type some characters again and take the whole available space
6. Rotate the device
7. Observe if buttons fit in the screen
8. Change the alignment to full width
9. Again observe if buttons fit in the screen

PR submission checklist:
